Sources/PointFree/Account/Account.swift

```diff
@@ -388,15 +388,38 @@ to consume our videos: an RSS feed that can be used with podcast apps!
                     )
                   ]
                 )
-                ])
-            ]
+                ]
+              )
+            ] + rssTerms(stripeSubscription: data.stripeSubscription)
           )
         ]
       )
       ])
   ]
 }
 
+private func rssTerms(stripeSubscription: Stripe.Subscription?) -> [Node] {
+  return stripeSubscription?.plan.interval == .some(.month)
```

Comment:
I've added this disclaimer copy to the bottom of the RSS link for when you are on a monthly plan.